content/pages/quickstart.md

```diff
@@ -25,7 +25,8 @@ This guide will lead you through creating a user site at `username.github.io`.
 {% data reusables.repositories.create_new %}
 1. Enter `username.github.io` as the repository name. Replace `username` with your {% data variables.product.prodname_dotcom %} username. For example, if your username is `octocat`, the repository name should be `octocat.github.io`.
    ![Repository name field](/assets/images/help/pages/create-repository-name-pages.png)
-{% data reusables.repositories.sidebar-settings %}
+2. Under your repository name, click **{% octicon "gear" aria-label="The gear icon" %} Settings**.
+   ![Repository settings button](/assets/images/help/pages/quickstart-settings-nav.png)
```

Comment:
The content of this image definitely makes more sense, and is a valuable update. It looks as though the brightness on your screen may have been turned down when you made the screenshot, and the resulting screenshot could be problematic from an accessibility standpoint. Would you mind creating a new screenshot with the same content, but ensuring that it is as bright as the original screenshot? For reference: 

![repo-actions-settings](https://user-images.githubusercontent.com/19994093/178698759-9fa21c88-4a99-4255-8298-98f5089e750d.png)